src/megatron/energon/flavors/webdataset/default_decoder_webdataset.py

```diff
@@ -56,6 +56,11 @@ def __init__(
         auto_decode: bool = True,
         image_decode: ImageDecoder = "torchrgb",
         ignore_decoder_errors: bool = False,
+        audio_clip_duration: int = 1,
```

Comment:
can we add a `video_decode: Literal['torch', 'AVData'] = "torch"` here? To select if to return the `AVData` or to decode the frames already given the options.

Reply:
done